modules/network-observability-ebpf-manager-operator.adoc

```diff
@@ -0,0 +1,38 @@
+// Module included in the following assemblies:
+//
+// * network_observability/observing-network-traffic.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="network-observability-ebpf-manager-operator_{context}"]
+= Working with the eBPF Manager Operator
+The eBPF Manager is a separate Operator that reduces the attack surface, and ensures compliance, security, and conflict prevention by managing all eBPF programs. Network observability can use eBPF Manager to load hooks. This eliminates the need to provide the eBPF Agent with privileged mode or additional Linux capabilities like `CAP_BPF` and `CAP_PERFMON`. The eBPF Manager with network observability is only supported on 64-bit AMD architecture.
+
+:FeatureName: eBPF Manager Operator with network observability
+include::snippets/technology-preview.adoc[]
+
+.Procedure
+. In the web console, navigate to *Operators* -> *Operator Hub*.
+. Install *eBPF Manager*.
+. Check *Workloads* -> *Pods* in the `bpfman` namespace to make sure they are all up and running.
+. Configure the `FlowCollector` custom resource, for example:
++
+[id="network-observability-flowcollector-configuring-ebpfmanager_{context}"]
+.Example `FlowCollector` configuration
+[source,yaml]
+----
+apiVersion: flows.netobserv.io/v1beta2
+kind: FlowCollector
+metadata:
+  name: cluster
+spec:
+  agent:
+    ebpf:
+      features:
+        - EbpfManager
+----
+
+.Verification
+. In the web console, navigate to *Operators* -> *Installed Operators*.
+. Click *eBPF Manager Operator* -> *All instances* tab.
+
+There should be a `BpfApplication` named `netobserv` and a pair of `BpfProgram` objects, one for Traffic Control (TCx) ingress and another for TCx egress, for each node. There might be more if you enable other eBPF Agent features.
```

Comment:
Should attach to the step to avoid text outside steps
```suggestion
. Click *eBPF Manager Operator* -> *All instances* tab.
+
There should be a `BpfApplication` named `netobserv` and a pair of `BpfProgram` objects, one for Traffic Control (TCx) ingress and another for TCx egress, for each node. There might be more if you enable other eBPF Agent features.
```